internal/cluster/validator.go

```diff
@@ -307,16 +307,17 @@ func (v *clusterValidator) sufficientMastersCount(c *clusterPreprocessContext) v
 		minMastersNeededForInstallation = common.AllowedNumberOfMasterHostsInNoneHaMode
 	}
 
-	knownHosts, ok := MapHostsByStatus(c.cluster)[models.HostStatusKnown]
-	if !ok { //if no known hosts exist, there is no sufficient master count
-		return boolValue(false)
+	hosts := make([]*models.Host, 0)
+	for k, v := range MapHostsByStatus(c.cluster) {
+		if k != models.HostStatusDisabled {
```

Comment:
I think that we should ignore also disconnected 
If we have 3 masters and 1 is in a disconnected state - it's not enough

Reply:
The cluster will be in insufficient state anyway, so it doesn't make much difference.

Reply:
I don't agree, we have other checks for disconnected.  Here we are just counting hosts regardless of state.